src/main/java/com/diffblue/javademo/nestedobjects/User.java

```diff
@@ -21,4 +21,12 @@ public boolean checkItemCost(Item item) {
   }
 
   private Order order;
+
+  public void setOrder(Order newOrder) {
+    order = newOrder;
```

Comment:
Is there a reason to not`return this;` in setters, so you can do things like `Foo.setBar(1).setBaz(2);`?

I may just be showing my JS bias - jQuery in particular made this chaining style very popular.